client/my-sites/domains/domain-search/domain-search.jsx

```diff
@@ -122,6 +126,15 @@ class DomainSearch extends Component {
 			'domain-search-page-wrapper': this.state.domainRegistrationAvailable,
 		} );
 		const { domainRegistrationMaintenanceEndTime } = this.state;
+
+		const domains = this.props.domains.filter( domain => domain.type !== 'WPCOM' );
```

Comment:
@matthusby FYI there is a handy constant for exactly this scenario in [lib/domains/constants](https://github.com/Automattic/wp-calypso/blob/master/client/lib/domains/constants.js#L7) An example usage can be found here: https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/domains/domain-management/edit/wpcom-domain.jsx#L32.

Reply:
Thanks! I just opened https://github.com/Automattic/wp-calypso/pull/30703 to address this :+1: